docs/get_started.rst

```diff
@@ -1,17 +1,345 @@
+Use the `cci project init` command from within a git repository to generate the initial version of a project's ``cumulusci.yml`` file.
+
+.. code-block:: console
+
+    $ cci project init
+    Name: MyRepoName
+    Package name: My Repo Name
+    Package namespace: mynamespace
+    Package api version [38.0]:
+    Git prefix feature [feature/]:
+    Git default branch [main]:
+    Git prefix beta [beta/]:
+    Git prefix release [release/]:
+    Test namematch [%_TEST%]:
+    Your project is now initialized for use with CumulusCI
+    You can use the project edit command to edit the project's config file
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: MyRepoName
+        package:
+            name: My Repo Name
+            namespace: mynamespace
+
+The newly created `cumulusci.yml` file is the configuration file for your project specific tasks, flows, and CumulusCI customizations.
+For more information regarding configuraiton, checkout our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can add and commit it to your git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+With your ``cumulusci.yml`` file committed, we now want to create a repository on GitHub for our new project and push our changes there.
+
+#. `Create a new repository <https://docs.github.com/en/free-pro-team@latest/articles/creating-a-new-repository>`_ on GitHub.
+#. At the top of your GitHub Repository's Quick Setup page, click the clipboard button to copy the remote repository URL.
+#. In your terminal, `add the URL for the remote repository <https://docs.github.com/en/free-pro-team@latest/articles/adding-a-remote>`_ to where your local repository will be pushed::
+
+    $ git remote add origin <remote_repository_url>
+
+#. Verify the remote was added successfullly with::
+
+    $ git remote -v
+
+#. `Push the changes <https://docs.github.com/en/free-pro-team@latest/github/using-git/pushing-commits-to-a-remote-repository>`_ in your local repository to GitHub::
+
+    $ git push -u origin master
+
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+
+Project Setup
+^^^^^^^^^^^^^
+#. Create a directory for your project to live in, and navigate to it::
+
+    $ mkdir mySalesforceProject; cd mySalesforceProject
+
+#. Initialize the directory as a git repository::
+
+    $ git init
+    Initialized empty Git repository in /Users/MrCCI/repos/mySalesforceProject/.git/
+
+#. Initialize the repository as a CumulusCI project. See `project initialization`_.
+
+
+Extracting Your Project's Metadata
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Next we need to extract your package metadata.
+Depending on what type of org your project is in, there are different methods for extracting the desired pieces of metadata.
+
+Production or Developer Edition Orgs
```

Comment:
I would suggest that we make these sections something like "Existing Scratch Org-based Project" and "Existing Developer Org-Based Project".

I'm not sure source tracking alone is sufficient to be helpful because in many cases existing project won't have SourceMember records. Projects already developed in scratch orgs shouldn't need to do a retrieve at all.

Reply:
@davidmreed - What are the groups of orgs that make sense to break this up into? I like the idea of (1) if you're developing in a scratch org, then you likely just need to run `sfdx force:source:pull`, and (2) if your project is in _any_ type of persistent org then we recommend this set of steps (currently Production & DE org section).

Let me know your thoughts.

Reply:
My thought is that if the user is already building in scratch orgs, we probably do not need to provide them with assistance with metadata capture - they already have their metadata in place. For those users, the most value we can provide is probably on how to establish their CCI project skeleton, move an existing SFDX org config into `orgs/dev.json`, and start using `dev_org`.

If the user is not already building in scratch orgs, the process they need to follow _does_ likely include metadata capture, regardless of the nature of the org where they are working. In that case, I think we need to cover two main topics:

- Metadata capture
- Org shape setup

Org shape setup will still be relevant for users that are building in a source-based, but sandbox- or developer org-oriented, process.